src/rmodels.c

```diff
@@ -3829,6 +3829,22 @@ RayCollision GetRayCollisionQuad(Ray ray, Vector3 p1, Vector3 p2, Vector3 p3, Ve
     return collision;
 }
 
+static void BuildPoseFromParentJoints(BoneInfo *bones, int boneCount, Transform *transforms) {
```

Comment:
Please, could you review the parentheses to be aligned, following raylib coding conventions?